### PR TITLE
Bug 1969719: Add trusted CA bundle to vsphere operators

### PR DESCRIPTION
**Pull request by jsafrane**
Use cluster's CA bundle when talking to vCenter.
/assign @gnufied

### CODE REVIEW

File: assets/csidriveroperators/vsphere/08_deployment.yaml [hunk 1]
```diff
@@ -50,6 +50,10 @@ spec:
           requests:
             memory: 50Mi
             cpu: 10m
+        volumeMounts:
+        - name: trusted-ca-bundle
+          mountPath: /etc/pki/ca-trust/extracted/pem
+          readOnly: true
```
Comment (gnufied) [inline assets/csidriveroperators/vsphere/08_deployment.yaml:56]: nit - configmaps are already readonly.

Comment (jsafrane) [reply assets/csidriveroperators/vsphere/08_deployment.yaml]: It's copied from our docs :-). I removed it.

**Comment by jsafrane** [comment]
```
Operator degraded (VSphereProblemDetectorDeploymentController_SyncError): VSphereProblemDetectorDeploymentControllerDegraded: panic caught:
VSphereProblemDetectorDeploymentControllerDegraded: asset: Asset(vsphere_problem_detector/06_deployment.yaml): Asset vsphere_problem_detector/06_deployment.yaml not found
```
Looking into this.

**Comment by jsafrane** [comment]
/retest
The bug above seems to be fixed, now I get just random flakes

**Comment by jsafrane** [comment]
/retest
?

**Pull request merged**